crates/bevy_render/src/camera/camera.rs

```diff
@@ -387,6 +389,8 @@ pub enum NormalizedRenderTarget {
     Window(NormalizedWindowRef),
     /// Image to which the camera's view is rendered.
     Image(Handle<Image>),
+    /// Texture View to which the camera's view is rendered.
+    TextureView(ManualTextureViewHandle),
```

Comment:
Same as my other suggestion
```suggestion
    /// Texture View to which the camera's view is rendered.
    /// Useful when the texture view needs to be created outside of Bevy, for example OpenXR. 
    TextureView(ManualTextureViewHandle),
```